---
focal_length: 0.013387
cam_height: 1.2573
fov_x: 130
fov_y: 103
resolution_x: 1920
resolution_y: 1080
